src/Terrain/MapLocation.java

```diff
@@ -40,19 +40,13 @@ public Commander getOwner()
     return owner;
   }
 
+  /**
+   * If the actual map state is changing, make sure to update the ownedProperties lists.<p>
+   * MapMaster.setOwner() is a convenient way to do so.
+   */
```

Comment:
Are there times we might call this where the actual map state is _not_ changing?

Reply:
Yup, `MapPerspective` does this. Quoth commit 1:
```
Move CO.ownedProperties management up to MapMaster

MapPerspective calls location.setOwner() a lot when it's creating
 the ephemeral capturable location objects.
I don't know for certain that that's the smoking gun on a crash
 in CommanderProductionInfo's constructor, but it seems messed up.
It also made the crash pretty well impossible to debug.
```